core/src/utilities/helpers/routing-helpers.js

```diff
@@ -597,6 +597,30 @@ class RoutingHelpersClass {
     };
     component.showAlert(alertSettings, false);
   }
+
+  // Adds and remove properties from searchParams
+  modifySearchParam(params, searchParams, paramPrefix) {
```

Comment:
```suggestion
  modifySearchParams(params, searchParams, paramPrefix) {
```
Please change to `modifySearchParams` which is more consistent to other params related functions. Thanks